src/coreclr/jit/compiler.h

```diff
@@ -6253,10 +6260,20 @@ class Compiler
 
     PhaseStatus fgFindOperOrder();
 
+#ifdef TARGET_WASM
+
     FlowGraphDfsTree* fgWasmDfs();
     PhaseStatus fgWasmControlFlow();
     PhaseStatus fgWasmTransformSccs();
 
+#ifdef DEBUG
+
+    void fgDumpWasmControlFlow();
+    void fgDumpWasmControlFlowDot();
+
+#endif
+#endif
```

Comment:
Nit: closing `// TARGET_WASM |  DEBUG` comments missing.

Also, are the newlines intentional? It is unusual to see newlines between `#ifdef` and what it is guarding.